pharus/component_interface.py

```diff
@@ -326,4 +354,5 @@ def dj_query_route(self):
     "file:image:attach": FileImageAttachComponent,
     "slider": BasicQuery,
     "dropdown-query": BasicQuery,
+    "form": InsertComponent,
```

Comment:
This comment is for the `basicquery` line above. (GH wouldn't let me comment directly on it)

We can simplify the implementation of it to just point to `FetchComponent` and move the `dj_query_route` into `FetchComponent` since it is overloaded by each child class.